packages/editor-ui/src/components/RunData.vue

```diff
@@ -433,7 +445,7 @@ export default mixins(
 				return defaults;
 			},
 			hasNodeRun(): boolean {
-				return Boolean(!this.isExecuting && this.node && this.workflowRunData && this.workflowRunData.hasOwnProperty(this.node.name));
+				return Boolean(!this.isExecuting && this.node && (this.workflowRunData && this.workflowRunData.hasOwnProperty(this.node.name) || this.hasPinData));
```

Comment:
`hasPinData` is independent of whether node has run. I would hoist `this.hasPinData` alongside `hasNodeRun` instead of nesting it inside.